### PR TITLE
3659 tempConverter

### PR DESCRIPTION
**Pull request by ShivanshTB**
https://dev.azure.com/thinkbridge-dev/psg_training/_backlogs/backlog/psg_training%20Team/Epics/?workitem=3659

### CODE REVIEW

File: src/app/temp-converter/temp-converter.component.ts [hunk 1]
```diff
@@ -3,13 +3,46 @@ import { Component, OnInit } from '@angular/core';
...
+  ngOnInit(): void {}
+  cel!: any;
+  cToF!: any;
+  feh!: any;
+  fToC!: any;
+  onInputCel() {
+    if (this.cel == 0) {
+      this.cToF = 0;
+    } else {
+      console.log(this.cel);
```
Comment (JuhiKthinkbridge) [inline src/app/temp-converter/temp-converter.component.ts]: Remove console.log from the final code. They are only used to debug your code while developing. These should not exist in the final running application as these may leak data to the end user.

Comment (ShivanshTB) [reply src/app/temp-converter/temp-converter.component.ts]: okay

File: src/app/temp-converter/temp-converter.component.ts [hunk 2]
```diff
@@ -3,13 +3,46 @@ import { Component, OnInit } from '@angular/core';
...
+  styleUrls: ['./temp-converter.component.css'],
 })
 export class TempConverterComponent implements OnInit {
+  constructor() {}
 
-  constructor() { }
-
-  ngOnInit(): void {
+  ngOnInit(): void {}
+  cel!: any;
```
Comment (mgudgila) [inline src/app/temp-converter/temp-converter.component.ts]: Try to mention types properly. 'any' type should be last resort. This applies for all variables.

Comment (ShivanshTB) [reply src/app/temp-converter/temp-converter.component.ts]: okay

**Pull request merged**